### PR TITLE
Live preview action

### PR DESCRIPTION
**Pull request by dsavchenko**

### CODE REVIEW

**Comment by dsavchenko** [comment]
I don't have a permission to set reviewer in this repo, so please review @volodymyrss

**Review by volodymyrss — APPROVED** [review]
I think it's ok actually, also not too complicated. 
Maybe test-live should be added automatically by bot?

**Comment by volodymyrss** [comment]
do you already have rights to merge @dsavchenko ? Anyway I merge, nice to have a look what it does. Let's add next changes in new PR as needed!

File: .github/workflows/live-preview.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+name: Preview PR on galaxy.odahub.fr
+on:
+  pull_request:
```
Comment (volodymyrss) [inline .github/workflows/live-preview.yaml:3]: it should also run on change label @dsavchenko

Comment (dsavchenko) [reply .github/workflows/live-preview.yaml:3]: There is pull_request: labeled event there, isn't it enough?

**Pull request merged**